tapis-project/tapis-ui
# Task/t UI 106  react query jobs submit

## Overview: ##

Jobs Submission uses react-query

## Related Github Issues: ##

* [TUI-106](https://github.com/tapis-project/tapis-ui/issues/106)

## Summary of Changes: ##

- tapis-api/jobs/submit
- tapis-hooks/jobs/useSubmit
- JobsLauncher uses react-query hooks

## Testing Steps: ##
1. npm start
2. Launch a job from an application
3. Switch to a different application - job submission state should reset
4. Change the exec system to something that's not the default. Upon submission you should see an error message
5. Switch to a different application - job submission state and error should reset

## UI Photos:

## Notes: ##
